custom/_legacy/pact/tasks.py

```diff
@@ -76,42 +70,12 @@ def eval_dots_block(xform_json, callback=None):
         notify_exception(None, message="PACT error evaluating DOTS block docid %s, %s\n\tTraceback: %s" % (xform_json['_id'], ex, tb))
 
 
-@periodic_task(run_every=crontab(hour="12", minute="0", day_of_week="*"))
-def update_schedule_case_properties():
-    """
-    Iterate through all pact patient cases in the domain and set schedule case properties if necessary.
-    """
-    case_ids = get_case_ids_in_domain('pact', 'cc_path_client')
-    # this is intentionally not user iter_docs since pact cases are *huge* and we want to use
-    # get_lite and only load one at a time.
-    for case_id in case_ids:
-        case = PactPatientCase.get_lite(case_id)
-        set_schedule_case_properties(case)
-
-
-def set_schedule_case_properties(pact_case):
-    """
-    Sets the required schedule case properties on the case if they are different from the current
-    case properties. See the README for more information.
-    """
-    SCHEDULE_CASE_PROPERTY_PREFIX = 'dotSchedule'
-    DAYS = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday']
-    current_schedule = pact_case.current_schedule
-    if current_schedule is not None:
-        to_change = {}
-        for day in DAYS:
-            case_property_name = '{}{}'.format(SCHEDULE_CASE_PROPERTY_PREFIX, day)
-            from_case = getattr(pact_case, case_property_name, None)
-            from_schedule = current_schedule[day]
-            if (from_case or from_schedule) and from_case != from_schedule:
-                to_change[case_property_name] = current_schedule[day]
-        if to_change:
-            case_block = CaseBlock(
-                case_id=pact_case._id,
-                update=to_change,
-            ).as_xml()
-            submit_case_blocks(
-                [ElementTree.tostring(case_block)],
-                'pact',
-                device_id=__name__ + ".set_schedule_case_properties",
-            )
+# Originally there was a task here:
+# @periodic_task(run_every=crontab(hour="12", minute="0", day_of_week="*"))
+# def update_schedule_case_properties():
+# This comment is a tombstone for someone later trying to resolve a mystery.
+# This task had been failing once a day for nearly a full year
+# starting on Mar 28, 2018 8:00:08 AM EDT.
+# When getting rid of the long-broken cases_get_lite couch design doc,
+# Danny decided to remove this failing task entirely,
+# which should have the same effect as running once a day and failing once a day.
```

Comment:
I think this comment should just be removed totally. its pretty close to commenting out code, and I don't see how this "tombstone" is going to help someone in the future figure out a bug that's already been failing for a year

Reply:
https://github.com/dimagi/commcare-hq/pull/23558